api/src/config/mod.rs

```diff
@@ -505,6 +524,12 @@ impl<'de> Deserialize<'de> for ApiAccess {
     }
 }
 
+impl FrontendConfig {
+    pub fn title(&self) -> Option<&String> {
+        self.title.as_ref()
+    }
+}
+
```

Comment:
As you made `title` public, there is no need for this impl block.

```suggestion

```

Reply:
Done.